src/db/groupCategories.ts

```diff
@@ -9,6 +9,8 @@ export const groupCategories = pgTable('group_categories', {
   id: uuid('id').primaryKey().defaultRandom(),
   name: varchar('name'),
   eventId: uuid('event_id').references(() => events.id),
+  userCanCreate: boolean('user_can_create').notNull().default(false),
+  userCanView: boolean('user_can_view').notNull().default(false),
```

Comment:
- Does that mean that `affiliation` would fall under the default? I.e. false, false? Or would it be the same as, for example, polis (view:true, create:false)? What would be the case of true, true: I.e. is that even allowed?

Reply:
Cant we achieve the same behaviour with userCanCreate alone?

Reply:
afilliation would fall under canview but can not create

Reply:
having just canCreate alone would make us have to guess when a user can view the groups, it just seems better to make things explicit rather than implicit

Reply:
Thanks for the explanation (did not know that comments are already visible even though the review was no submitted :D. See updated comment.

Reply:
i think affiliation would be the same as polis. this feature just gives us more control on a possible situation in the future. if we only had one boolean we would have to guess what it meant in some cases, this just makes it more understandable